src/TableRow.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+function TableRow(props) {
+  const [highlight, setHighlight] = useState(false);
+
+  function highlightWhenClicked() {
+    setHighlight(!highlight);
+  }
+
+  function handleClick() {
+    props.searchId(props.customer.id);
+  }
+
+  return (
+    <tr
+      key={props.customerIndex}
```

Comment:
I think this is actually unnecessary 🙂 Try removing it and look to see if you get a warning in the dev tools console.